package exceptions;

public class TaskDoneException extends Exception {
    public TaskDoneException() {};

    public String toString() {
        return "â˜¹ OOPS!!! Task is already completed!";
    }
}
